vision_zero/schools.py

```diff
@@ -31,25 +32,35 @@
 
```

Comment:
While these lines are being run. They don't actually affect how the airflow script is being run, since that doesn't run the part under `if __name__ == '__main__':`. The DAG is actually already pulling credentials from Airflow's configurations. Just move these lines pulling from Joven's home folder to the `main` block so importing the script in Airflow won't fail if Joven's home folder gets deleted.

https://github.com/CityofToronto/bdit_data-sources/blob/0b77909b07ed0c8c119c93fd8a4253fdad3954ee/vision_zero/schools.py#L28-L31

Reply:
done